README.md

```diff
@@ -24,6 +24,7 @@ cd manifests/xgboost-job/xgboost-operator
 kubectl create namespace kubeflow
 kustomize build base | kubectl apply -f -
 ```
+Note: for windows user, you might have invalid path error when you clone kubeflow/manifests repository, please use WSL instead
```

Comment:
Add link to WSL?

Also remove “note :”

Why would they have issue git cloning the repo?

Reply:
If I use powershell to do the git clone for kubeflow/manifest repository will have following error
error: invalid path 'tests/tests/legacy_kustomizations/cert-manager-kube-system-resources/test_data/expected/rbac.authorization.k8s.io_v1beta1_role_cert-manager-cainjector:leaderelection.yaml'
fatal: unable to checkout working tree
warning: Clone succeeded, but checkout failed.
You can inspect what was checked out with 'git status'
and retry with 'git restore --source=HEAD :/'

I think this is a common error for windows user

Reply:
There are some other ways to solve this problem for Windows user, some method might be a little bit easier if the user haven't installed WSL in their computer. But I guess most developer who use Windows has already installed WSL in their computer, then use WSL seems the easiest solution

Reply:
I can also put a link for other solution if the windows user don't want to install WSL.
https://brendanforster.com/notes/fixing-invalid-git-paths-on-windows/

Reply:
I am not sure if it makes sense to add this specific note that's git and OS specific. It sounds like this should be something a user would be figure out on their own by Googling around.

Reply:
I agree